tmk_core/chibios.mk

```diff
@@ -385,8 +385,14 @@ ifndef TEENSY_LOADER_CLI
     endif
 endif
 
+ifeq ($(strip $(MCU_FAMILY)),MIMXRT1062)
+    TEENSY_LOADER_CLI_MCU = imxrt1062
```

Comment:
Isn't there a better way to do this?  

Such as add a folder to `/platform/chibiOS` for the teensy 4?

Reply:
Yes, that seems to work. I’ll update the PR in a bit.

Is there any documentation available for how QMK’s build system includes files? I.e. in which order, which files are discovered from where, etc.?

Reply:
Done